cmd/wallet-web/src/pages/MultipleQuery.vue

```diff
@@ -5,266 +5,294 @@
-                    <div class="cardDetailsL">
-                      <md-icon class="md-size-4x">{{ card.icon }}</md-icon>
-                    </div>
-                    <div class="cardDetailsR">
-                      <p>{{ card.description }}</p>
-                      <div v-if="card.body">
-                        The verifier can only access below information from your credential.
-                        <div v-for="(subj, skey) in card.body" :key="skey">
-                          <div v-if="displayContent(skey)" class="md-caption">
-                            <b>{{ skey.replace('.', ' ') }} </b>: {{ subj }}
-                          </div>
-                        </div>
-                      </div>
-                    </div>
-                  </div>
-                </div>
-              </div>
-            </transition>
-          </li>
-        </ul>
-      </md-card-content>
-
-      <div v-if="showShareCredential" class="md-layout md-alignment-center-center">
-        <p class="md-body-1">
-          By clicking Agree you will be sharing a unique identifier to
-          <b style="color: #2e7d32">{{ requestOrigin }}</b
-          >, the Credential content, and your digital signature.
-          <a href="https://www.w3.org/TR/vc-data-model/#proofs-signatures" target="_blank"
-            >Learn more</a
-          >
-        </p>
-
-        <md-button
-          id="share-credentials"
-          class="md-button md-info md-square md-theme-default md-large-size-100 md-size-100 col"
-          style="background-color: #29a329 !important"
-          @click="share"
-        >
-          Agree
-        </md-button>
-        <md-button
-          id="cancelBtn"
-          style="margin-left: 5px; background-color: #9d0006 !important"
-          class="md-cancel-text"
-          @click="cancel"
-        >
-          Cancel
-        </md-button>
+        <button id="cancelBtn" class="btn-outline" @click="cancel">
+          {{ $t('CHAPI.Share.decline') }}
+        </button>
+        <button id="share-credentials" class="btn-primary" @click="share">
+          {{ $t('CHAPI.Share.share') }}
+        </button>
       </div>
     </div>
   </div>
 </template>
 <script>
 import { CredentialManager } from '@trustbloc/wallet-sdk';
-import {
-  CHAPIEventHandler,
-  extractPresentationExchangeReasons,
-  extractQueryReasons,
-  flatCredentialSubject,
-  normalizeQuery,
-  getVCIcon,
-  getCredentialType,
-} from './mixins';
+import { normalizeQuery, getCredentialType, getCredentialDisplayData } from './mixins';
 import { mapGetters } from 'vuex';
-import Governance from './Governance.vue';
+import Spinner from '@/components/Spinner/Spinner.vue';
+import CredentialPreview from '@/components/CredentialPreview/CredentialPreview.vue';
+import credentialDisplayData from '@/config/credentialDisplayData';
 
-const nonDisplayContent = ['id', 'type'];
+const filterBy = ['IssuerManifestCredential', 'GovernanceCredential'];
```

Comment:
Query will never return these 2 types, should be ok to remove I guess.
Add a TODO and remove it in next PR.